game-dev/ressources.md

```diff
@@ -1,4 +1,10 @@
-# Ressources pour game-dev
+---
+layout: post
+author: Le serveur Not a Name
+date: 2021-09-04
+last_update: 2023-04-12
```

Comment:
```suggestion
last_update: 2023-03-12
```

Reply:
https://github.com/NotANameServer/Not-a-Hub/commits/master/game-dev/ressources.md